index.html

```diff
@@ -72,9 +72,7 @@
                     <li class="nav-item">
                         <a class="nav-link  font-weight-bold nav-items" href="team.html">Team</a>
                     </li>
-                    <li class="nav-item">
-                        <a class="nav-link  font-weight-bold nav-items" href="#footer">Contact</a>
-                    </li>
+                    
```

Comment:
Any reason to change the order ?

Reply:
yes actually if we aligned in the right then the dropdown menu's navigation will no more be aligned like this:
![image](https://user-images.githubusercontent.com/56486309/113034010-44880000-91af-11eb-93c6-681ef04bcd6c.png)

Reply:
Okay